.changeset/clean-ravens-rest.md

```diff
@@ -0,0 +1,5 @@
+---
+"@salt-ds/core": patch
+---
+
+Fixed multiline input not collapsing when value is reduced
```

Comment:
The actual issue is the multiline input not collapsing when it's controlled API is used.